vxquery-core/src/main/java/org/apache/vxquery/exceptions/SystemException.java

```diff
@@ -19,10 +19,10 @@
 import java.text.MessageFormat;
 import java.util.Arrays;
 
-import org.apache.hyracks.algebricks.common.exceptions.AlgebricksException;
+import org.apache.hyracks.api.exceptions.HyracksDataException;
 import org.apache.vxquery.util.SourceLocation;
 
-public class SystemException extends AlgebricksException {
+public class SystemException extends HyracksDataException {
```

Comment:
After our discussions, I think we should create an issue to address this in a different change.